apps/system-e2e/src/support/session.ts

```diff
@@ -157,11 +157,19 @@ export async function judicialSystemSession({ browser }: { browser: Browser }) {
   const context = await browser.newContext()
   const page = await context.newPage()
   const authUrlPrefix = urls.authUrl
-  await ensureCognitoSessionIfNeeded(
-    page,
-    JUDICIAL_SYSTEM_HOME_URL,
-    authUrlPrefix,
-  )
-  await page.close()
+  // Retry a few times in case login fails due to deployment not being stable
```

Comment:
If you really need retrying, why not move it into the `ensureCognitoSessionIfNeeded` to benefit all unstable tests?

Reply:
I mostly just wanted to see if this would fix the issue before introducing it to every test that's run in other projects :)